packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -39,11 +46,32 @@
     $: isImplicitAccount = isImplicitAccountOutput(selectedOutput.output as CommonOutput)
     $: accountId = isAccountOutput(selectedOutput) ? (selectedOutput?.output as AccountOutput)?.accountId : null
     $: address = accountId ? getBech32AddressFromAddressTypes(new AccountAddress(accountId)) : null
+    $: hasStakedFeature = hasOutputStakedFeature(selectedOutput)
 
     function onExplorerClick(): void {
         const url = `${explorerUrl}/${ExplorerEndpoint.Output}/${selectedOutput.outputId.toString()}`
         openUrlInBrowser(url)
     }
+
+    function hasOutputStakedFeature(output: OutputData): boolean {
+        return (
+            isAccountOutput(output) &&
+            (output.output as AccountOutput).features?.some((feature) => feature.type === FeatureType.Staking)
+        )
+    }
+
+    function getStakedAmount(): string {
+        let amount = '0'
+        if (isImplicitAccount) return amount + 'i'
```

Comment:
what is `i`? the unit? we should not hardcode any unit, it should be taken from the network you are in